wagtail/contrib/forms/templates/wagtailforms/list_submissions.html

```diff
@@ -20,11 +15,10 @@
             </tr>
         </thead>
         <tbody>
+            {% trans "Select document" as checkbox_aria_label %}
```

Comment:
This looks like the wrong text.

Reply:
I could not come up with a name so named it document, can we name it as entry ?

Reply:
I have a pull request with these changes